packages/fuselage/src/components/Slider/Slider.tsx

```diff
@@ -0,0 +1,100 @@
+import type { AriaAttributes } from 'react';
+import React, { useRef } from 'react';
+import { useNumberFormatter, useSlider } from 'react-aria';
+import { useSliderState } from 'react-stately';
+
+import { SliderHead } from './SliderHead';
+import { SliderThumb } from './SliderThumb';
+import { SliderTrack } from './SliderTrack';
+
+type SliderProps<T = number | number[]> = AriaAttributes & {
```

Comment:
If `multiThumb` is about having *two* thumbs, we should use a stricter constraint for `T`:

```suggestion
type SliderProps<T = number | [min: number, max: number]> = AriaAttributes & {
```

Reply:
totally agree. But this type is also what the lib expects and I had trouble when I typed it like you suggested. 
could you help with this?